build.gradle

```diff
@@ -209,8 +209,8 @@ task importDataFull(type: com.marklogic.gradle.task.MlcpTask) {
   input_file_type = "delimited_json"
   input_compressed = project.property("importDataFileIsCompressed")
   input_compression_codec = "gzip"
-  output_permissions = project.property("mlDataPermissions")
```

Comment:
if mlDataPermissions is no longer used, can it be removed from the properties files? Or is it used elsewhere implicitly by mlGradle? It is also placed into appConstants.mjs as a token replacement, but then the constant is never used.

Reply:
Good catch.  It does look like it is becoming obsolete.  In this PR, I leave open the possibility of which way to permission content: see no. 1 in the [Tenant Limitations and Warnings](https://github.com/project-lux/lux-marklogic/blob/updated-data-slices/docs/lux-backend-deployment.md#tenant-limitations-and-warnings).  I'm good with removing now or later (if we can remember).

Reply:
Gotcha. I'm ok with leaving it for now. If we decide to lean fully into permission setting via documentTransforms.sjs then let's try to remember to clean it up